src/early-late-bound.md

```diff
@@ -89,7 +89,7 @@ Moreover, a reference to a late-bound lifetime is written `^0.a`:
 - The `0` is the index; it identifies that this lifetime is bound in the
   innermost binder (the `for`).
 - The `a` is the "name"; late-bound lifetimes in rustc are identified by a
-  "name" -- the [`BoundRegion`] struct. This struct can contain a
+  "name" -- the [`BoundRegionKind`] enum. This enum can contain a
```

Comment:
It seems to me that we should still be linking to `BoundRegion` – it's just a struct now, rather than an enum.

Reply:
I think it's more straightforward to use `BoundRegionKind` like `ExprKind` or similar, since we use it `Kind` thing mainly.

Reply:
Given that `BoundRegion` is only a wrapper around `BoundRegionKind`, and that we talk about what *kind* of name it can be in this paragraph, I think linking to `BoundRegionKind` makes more sense

Reply:
In the future `BoundRegion` may have more fields than just `kind`, so wouldn't we want to link people to that?

Reply:
Oops, I was too late 😄

Reply:
I'd argue that if this is the case this chapter will need updating anyway